lua/lazyvim/plugins/extras/lang/go.lua

```diff
@@ -14,13 +14,31 @@ return {
     "neovim/nvim-lspconfig",
     opts = {
       servers = {
-        gopls = {
-          settings = {
+        gopls = function(_, opts)
```

Comment:
This should always be a table. Did you mean to put it under `opts.setup`?

Reply:
Oh, yes! It should be insider setup. Fixed that.